chapters/en/chapter12/8.mdx

```diff
@@ -0,0 +1,142 @@
+# Production Inference with TGI 
+
+# Text Generation Inference (TGI)
+
+Text Generation Inference (TGI) is a toolkit developed by Hugging Face for deploying and serving Large Language Models (LLMs). It's designed to enable high-performance text generation for popular open-source LLMs. TGI is used in production by Hugging Chat - An open-source interface for open-access models.
+
+## Why Use Text Generation Inference?
+
+Text Generation Inference addresses the key challenges of deploying large language models in production. While many frameworks excel at model development, TGI specifically optimizes for production deployment and scaling. Some key features include:
+
+- **Tensor Parallelism**: TGI's can split models across multiple GPUs through tensor parallelism, essential for serving larger models efficiently.
+- **Continuous Batching**: The continuous batching system maximizes GPU utilization by dynamically processing requests, while optimizations like Flash Attention and Paged Attention significantly reduce memory usage and increase speed.
+- **Token Streaming**: Real-time applications benefit from token streaming via Server-Sent Events, delivering responses with minimal latency.
+
+## How to Use Text Generation Inference
+
+### Basic Python Usage
```

Comment:
we should show how to turn a TGI instance on as well - just via the docker image